packages/admin-web-angular/src/app/@shared/warehouse/forms/warehouse-manage-tabs/details/warehouse-manage-tabs-details.component.ts

```diff
@@ -176,6 +181,7 @@ export class WarehouseManageTabsDetailsComponent
 			preferRestrictedCarriersForDelivery: [false],
 			carriersIds: [[]],
 			ordersShortProcess: [false],
+			inStoreMode: [],
```

Comment:
@sunk0 I am not sure, but could be this should be actually `[false]`, not `[]`?

Reply:
@evereq actually it works if you want i can redone

Reply:
It works, but maybe it did not set the value to false or? Anyway, leave it for now... I merged PR already, we will retest later